jira/jira-v1.asciidoc

```diff
@@ -0,0 +1,106 @@
+
+
+[options="header",cols="1m,1,1m,1,1"]
+|=======================
+|Request type|Endpoint|HTTP operation |Query parameter |Description 
+
+|Root Request |/rest/api/2/search|GET    |fields=assignee,issuetype,priority,project,reporter,status,summary,updated,attachment&startAt=1&maxResults=50|Returns all jira issues. Each issue contains its list of attachments metadata
+|Root Request|/rest/api/2/project/search|GET|description,lead,issueTypes,url,projectKeys,permissions,insight|Return all jira projects. Each jira project will index as seperate solr document.
+|Child Request |/rest/api/2/issue/`${LW_PARENT_DATA_KEY}`/comment|GET|startAt=0&maxResult=50| Returns all comments. The request requires the 'issue Id'. Internally, the plugin replaces the variable `${LW_PARENT_DATA_KEY}` with the 'issue id', which value is extracted from the 'issue object' (root object) by setting the parentDataKey property.
+|Child Request |/rest/api/2/issue/`${LW_PARENT_DATA_KEY}`/worklog | GET |startAt=0&maxResult=50 | Returns all worklogs. The request requires the 'issue Id'. Internally, the plugin replaces the variable `${LW_PARENT_DATA_KEY}` with the 'issue id', which value is extracted from the 'issue object' (root object) by setting the parentDataKey property.
+|Child Request |/rest/api/2/attachment/content/`${LW_PARENT_DATA_KEY}`|GET|-|Download the attachment content. The request requires the 'attachment Id'. Internally, the plugin replaces the variable `${LW_PARENT_DATA_KEY}` with the 'attachment id', which value is extracted in two steps: 1) setting prop parentNestedDataPath= fields.attachment[*] to extract the attachment metadata list from the issue object 2) setting prop parentDataKey=id to extract the 'id' from each attachment metadata. The download request will be performed per each attachment metadata found 
+|=======================
+
+
+== Terminology
+
+The following terms are provided as a reference.
+
+[options="header",cols="1s,1"]
+|=======================
+
+|Term|Description
+|Service Endpoints|The list of service endpoints from which the data is retrieved. Each service endpoint configures a root endpoint request.
+|Root Request|The type of request to retrieve a list of root data objects.
+|Child Request|The type of request to retrieve additional information for the root data objects. The child requests will be performed per each root data object.
+|Root Response Mapping|Defines the mapping between the response and data objects to be indexed.
+|Child Response Mapping |Defines the mapping between the child response and child data objects to be indexed.
+|Data Path|The path to access a specific data object within a response. For example, to access a list of elements named with key `objects`, the DataPath would be `objects`. If not provided, the entire response body will be indexed. This property accepts JsonPath expressions e.g.objects, objects[*] or $.objects[*]
```

Comment:
The JsonPath expression format is incorrect -> objects, objects[] or $.objects[]
![image](https://github.com/user-attachments/assets/cf0cb570-cf83-4736-b8cf-97cdffe19a6c)
.
Enclose the JsonPath expressions with backtick to show them correctly ->
e.g.`objects`, `objects[*]` or `$.objects[*]`